src/main/java/racingcar/domain/strategy/MoveStrategy.java

```diff
@@ -0,0 +1,7 @@
+package racingcar.domain.strategy;
+
+public interface MoveStrategy {
```

Comment:
```suggestion
@FunctionalInterface
public interface MoveStrategy {
```

MoveStrategy인터페이스 👍 이런 형태의 인터페이스를 함수형 인터페이스라고 부른답니다. @FunctionalInterface가 무엇인지 찾아보고 공부해 보세요.